src/components/LoansByCategory/FeaturedHeroLoan.vue

```diff
@@ -269,22 +310,45 @@ export default {
 				ids: this.featuredCategoryIds,
 			};
 		},
-		result({ data, loading }) {
-			if (loading) {
-				this.loading = true;
-			} else {
-				this.loan = _get(
-					_filter(data.lend.loanChannelsById, ['id', this.featuredCategoryIds[0]]),
-					'[0].loans.values[0]'
-				);
-
-				this.loanChannel = _get(
-					_filter(data.lend.loanChannelsById, ['id', this.featuredCategoryIds[0]]),
-					'[0]'
-				);
-
-				this.loading = false;
-			}
+		preFetch(config, client) {
+			return client.query({
+				query: featuredLoansQuery,
+				variables: {
+					ids: featuredCategoryIds
+				},
+			}).then(() => {
+				// eslint-disable-next-line max-len
+				return client.query({ query: experimentAssignmentQuery, variables: { id: 'featured_hero_loan_fundraising_thermometer' } });
+			});
+		},
+		result({ data }) {
```

Comment:
did you check to see if data.general.fundraisingThermometerExperiment is here... you may be able to skip the extra readQuery below.

Reply:
@mcstover I did check `data.general.fundraisingThermometerExperiment`and it returns the raw setting value:

```
{key: "uiexp.featured_hero_loan_fundraising_thermometer", value: ""{\"name\":\"FeaturedHeroLoanFundraisingThermomete…istribution\":{\"variant-a\":0,\"variant-b\":1}}"", __typename: "Setting"}
```

Reply:
oh right... thanks for verifying